src/components/page-deliveries/page-deliveries.tsx

```diff
@@ -36,23 +36,8 @@ const FoodChoices: FunctionalComponent<{
   </div>
 );
 
-const OrderDetailDisplay: FunctionalComponent<{
+const OrderDetailsDisplay: FunctionalComponent<{
```

Comment:
These two have a different layout and text copy. If you've consolidated the API and made all order data have the same structure we can consolidate the `OrderInfo`/`OrderDetail` types and rename these components, but we still need to change it based on whether the given location is selected or not. https://www.figma.com/file/v6kRIcxrdXBJZ4ipvJmh88/MVP-and-Style-Guide?node-id=8%3A0

Reply:
the only different is they potentially won't have the `location` attr - so that's a bit different

Reply:
b47bf31 should restore the different components - and moves `reports` into `OrderInfo`

Reply:
Not sure I'm following.  My concern is these two components were different in their content, not just in the type they used. I had thought you augmented the API and we don't need the `OrderInfo` type anymore, maybe I misunderstood, but we still need both of these display components. One is for the main page and the other when a location is selected.